components/html-block/README.md

```diff
@@ -0,0 +1,23 @@
+# HTML Block
+
+## d2l-html-block
+
+The `d2l-html-block` element is a web component for displaying user-authored HTML. It includes styles for headings, lists, anchors and other elements.  In addition, it provides integration with MathJax for rendering MathML.
+
+```html
+<script type="module">
+  import '@brightspace-ui/core/components/html-block/html-block.js';
+</script>
+
+<d2l-html-block html="..."></d2l-html-block>
+```
+
+**Properties:**
+
+| Property | Type | Description |
+|--|--|--|
+| `html` | String | HTML to be rendered |
+
+## Future Enhancements
+
+Looking for an enhancement not listed here? Create a GitHub issue!
```

Comment:
You could list replace strings here since it's a potential future enhancement we're already aware of?

Reply:
I've had some discussion with Eric about replace-strings and I think we're in agreement that all the replace strings should be handled server-side.  The HM API's can return both the replaced (for display) and unreplaced (for editing) versions of the HTML. So, consumers using this component would be feeding it HTML that has replacements already done.

Reply:
Gotcha -- makes sense, similar to HTML filtering. 👍